arroyo/backends/kafka/consumer.py

```diff
@@ -628,6 +628,11 @@ def close(self, timeout: Optional[float] = None) -> None:
     def closed(self) -> bool:
         return self.__state is KafkaConsumerState.CLOSED
 
+    @property
+    def member_id(self) -> str:
+        member_id: str = self.__consumer.memberid()
+        return member_id
```

Comment:
Does this not work?

```suggestion
        return self.__consumer.memberid()
```

Reply:
> Does this not work?

the type checker doesn't like it because `self.__consumer.memberid()` returns `Any`